charts/README.md

```diff
@@ -41,6 +41,7 @@ $ helm uninstall karmada -n karmada-system
 ```
 
 The command removes all the Kubernetes components associated with the chart and deletes the release.
+> **Note**: Still need to delete RBAC resources used by pre-install-job manually, such as serviceAccount/karmada-pre-job, clusterRole/karmada-pre-job, clusterRoleBinding/karmada-pre-job.
```

Comment:
```suggestion
> **Note**: There are some RBAC resources that are used by the `preJob` that can not be deleted by the `uninstall` command above. You might have to clean them manually with tools like `kubectl`.  You can clean them by commands:
```bash
$ kubectl delete serviceaccounts xxxx -n karmada-system  (I'm not sure if it is the namespace)
$ kubectl delete xxx
$ kubectl delete namespaces xxxxx 
```

```

We should give a clearly resource list here, not just like `such as`.

Reply:
Done